contrib/builder/deb/amd64/debian-wheezy/Dockerfile

```diff
@@ -4,6 +4,10 @@
 
 FROM debian:wheezy-backports
 
+# allow replacing httpredir mirror
+ARG APT_MIRROR=httpredir.debian.org
+RUN sed -i s/httpredir.debian.org//g /etc/apt/sources.list
```

Comment:
$APT_MIRROR variable is missing. should it be

```RUN sed -i s/httpredir.debian.org/$APT_MIRROR/g /etc/apt/sources.list``` ?

That is what you have on `contrib/builder/deb/armhf/debian-jessie/Dockerfile`